data/json/bionics.json

```diff
@@ -849,7 +845,7 @@
     "name": { "str": "Air Filtration System" },
     "description": "Surgically implanted in your trachea is an advanced filtration system.  If toxins, or airborne diseases find their way into your windpipe, the filter will attempt to remove them.",
     "occupied_bodyparts": [ [ "torso", 4 ], [ "mouth", 2 ] ],
-    "env_protec": [ [ "mouth", 7 ] ],
+    "env_protec": [ [ "mouth", 9 ] ],
```

Comment:
What's the logic here?

Reply:
You need certain thresholds to protect you from various gases, smoke.
7 was basically useless, 9 should be better.
I can't find actual thresholds for environmental dangers though.